Code/Framework/AzToolsFramework/AzToolsFramework/Input/QtEventToAzInputManager.cpp

```diff
@@ -284,13 +295,25 @@ namespace AzToolsFramework
     {
         auto systemCursorChannel =
             GetInputChannel<AzFramework::InputChannelDeltaWithSharedPosition2D>(AzFramework::InputDeviceMouse::SystemCursorPosition);
+        auto cursorXChannel =
```

Comment:
Nit-pick: cursorXChannel -> movementXChannel

(I know with this change we're now using the cursor position delta to approximate mouse movement values, but the channel itself being stored here is the movement one not the cursor one).